tests/integration/components/bs-form/element-test.js

```diff
@@ -589,4 +589,4 @@ test('shows help text if available', function(assert) {
 
   assert.ok(find('input').getAttribute('aria-describedby'), 'control has aria-describedby attribute');
   assert.equal(find('input').getAttribute('aria-describedby'), find(helpTextClass).getAttribute('id'), 'aria-describedby matches id');
-});
\ No newline at end of file
+});
```

Comment:
There should be a couple of new tests to cover the changes here, specifically the newly introduced classes (`col-form-label`, `form-check`, `form-check-label`, `form-check-input`). @Rapid0o can you add those? See the other tests for examples, sometimes there are separate tests for bs3/bs4, sometimes they are just diffentiated by using a test helper function to get the expected class name...